Guide for developers (#165)
flanksource/canary-checker: README.md

| endpoint |  | string | Yes |
| thresholdMillis |  | int64 | Yes |

### Guide for Developers

> This guide aims to provide the step by step process for creating your local setup with the canary-checker

Reviewer: ```suggestion
This guide provides a step-by-step process for creating your local setup with the canary-checker
```

> [Dev Guide](docs/dev-guide.md)

Reviewer: ```suggestion
[Dev Guide](docs/dev-guide.md).
```
